setup.py

```diff
@@ -1,6 +1,7 @@
 import sys
 from setuptools import setup, find_packages
 from setuptools.command.test import test as TestCommand
+from oval_graph.version import __version__
```

Comment:
This will import your installed `oval_graph`'s version attribute. You probably need to add the `setup.py`'s directory to the `sys.path`. Check out [another setup.py](https://github.com/matejak/imreg_dft/blob/master/setup.py) for inspiration.